README.md

```diff
@@ -844,174 +908,201 @@ $ docker run -d -it -p 7000:8000 -p 7001:8001 -p 7002:8002 \
 
 Finally, disconnect VMs from the swarm running the following command on each VM:
 
-```
+```bash
 docker swarm leave --force
 ```
+
 If the process is successful, a message saying the node has left the swarm will be displayed.
 
-# Image Tag
+## Image Tag
 
 The `marklogic` image tags allow the user to pin their applications to images for a specific release, a specific minor release, a specific major release, or the latest release of MarkLogic Server
 
-## `{ML release version}-{platform}-{ML Docker release version}`
+### `{ML release version}-{platform}-{ML Docker release version}`
 
 This tag points to the exact version of MarkLogic Server, the base OS, and the supporting scripts version. This allows an application to pin to a very specific version of the image. The image will not be updated without incrementing either the MarkLogic Sever version or the version of the supporting scripts.
 
 e.g. `11.0.3-centos-1.0.2` is the MarkLogic Server 11.0.3 release, CentOS, version 1.0.2 of the docker scripts.
 
-## `latest-xx.x`
+### `latest-xx.x`
 
 This tag points to the latest patch release of a specific minor version of MarkLogic Server on CentOS. The image will contain the latest docker supporting scripts and OS patches.
 
 e.g. `latest-11.0` is the latest patch release of MarkLogic Server 11.0 (11.0.0, 11.0.1, etc.).
 
 For MarkLogic 10, because the numbering scheme was changed, the maintenance release is equivalent to the minor release in MarkLogic 11. Use the `latest-10.0-x` tag to pin to a specific maintenance release of MarkLogic 10.
 
-## `latest-xx`
+### `latest-xx`
 
 This tag points to the latest minor and patch release of a specific major version of MarkLogic Server on CentOS. The image will contain the latest supporting scripts and OS patches.
 
 e.g. `latest-11` is the latest patch release of the latest minor release of MarkLogic Server 11 (11.0.0, 11.0.1, 11.1.0, 11.1.1, etc.)
 
 For MarkLogic 10, because the numbering scheme was changed, the maintenance release is equivalent to the minor release in MarkLogic 11. Use the `latest-10` tag to get the latest patch release of the latest maintenance release MarkLogic 10.
 
-## `latest`
+### `latest`
 
 This tag points to the latest minor, patch, and major release of MarkLogic Server on CentOS. The image will contain the latest supporting scripts and OS patches.
 
 It will pull the latest image and can cross patch, minor or major release numbers (11.0.0, 11.0.1, 11.1.0, 11.1.1, 12.0.0, etc.)
 
+**Note: The 'latest' images should not be used in production.**
+
+## Container Runtime Detection
+
+Since MarkLogic 11.2, MarkLogic is able to detect on which container runtime it is running on.
+
+### Docker Engine
+
+When running on Docker Engine the following entry will show up in the ErrorLogs.txt:
+
+`2024-03-15 08:27:36.136 Info: MarkLogic Server is running in a container using Docker runtime. A maximum of 1152 huge pages will be used if available`
```

Comment:
Change the log message examples to 
`Info: MarkLogic Server is running in a container using Docker runtime. A maximum of <x> huge pages will be used if available`

Reply:
Add

"Where <x> is calculated as described in the [Configuring HugePages](configuring-hugepages) section."

Reply:
Do this for each of the container type sections.